src/Faker/Provider/Internet.php

```diff
@@ -120,7 +130,15 @@ public function domainWord()
     {
         $lastName = $this->generator->format('lastName');
 
-        return strtolower(static::transliterate($lastName));
+        $lastName = strtolower(static::transliterate($lastName));
+        $lastName = rtrim($lastName, '.');
+
+        // fallback in case transliterate() doesn't support the language and removed all letters
```

Comment:
I don't like that fallback. If the locale doesn't manage to create username, it's the user's choice to replace it by another locale. 

In case the `$lastName` is empty, I prefer that we throw an exception here. It will break code, but at least that'll let users deal with it the way they want.